views/js/review/plugins/content/item-answer/item-answer.js

```diff
@@ -13,7 +13,7 @@
  * along with this program; if not, write to the Free Software
```

Comment:
CI unit test for unrelated `review/plugins/navigation/review-panel/plugin/test.html` is failing.
It was the same in previous PR: https://github.com/oat-sa/extension-lti-test-review/actions/runs/8049655532/job/22035921240?pr=142

Locally for me it succeeds, so I don't plan on fixing it.